avocado-setup.py

```diff
@@ -297,51 +218,17 @@ def get_repo(repo, basepath, install=False):
     repo_name = repo.split('/')[-1].split('.')[0]
     repo_path = os.path.join(basepath, repo_name)
     if os.path.isdir(repo_path):
-        logger.info("Updating the repo: %s in %s", repo_name, repo_path)
         cmd = "cd %s;git remote update;git merge origin master" % repo_path
-        err_str = "Failed to update %s repository:" % repo_name
-        try:
-            status, output = commands.getstatusoutput(cmd)
-            if status != 0:
-                logger.error("%s %s", err_str, output)
-                sys.exit(1)
-            logger.debug("%s", output)
-        except Exception, error:
-            logger.error("%s %s ", err_str, error)
-            sys.exit(1)
+        helper.runcmd(cmd,
+                      info_str="Updating the repo: %s in %s" % (repo_name, repo_path),
+                      err_str="Failed to update %s repository:" % repo_name)
     else:
-        err_str = "Failed to clone %s repository:" % repo_name
         cmd = "cd %s;git clone %s %s" % (basepath, repo, repo_name)
-        try:
-            status, output = commands.getstatusoutput(cmd)
-            if status != 0:
-                logger.error("%s %s", err_str, output)
-                sys.exit(1)
-            logger.debug("%s", output)
-        except Exception, error:
-            logger.error("%s %s", err_str, error)
-            sys.exit(1)
+        helper.runcmd(cmd,
+                      debug_str="Cloning the repo: %s" % repo_name,
```

Comment:
@sathnaga why is this `debug_str` and not `info_str` ?

Reply:
Also, it could be `Cloning the repo: %s in %s % (repo_name, repo_path)`, similar to the `updating` section.

Reply:
@narasimhan-v makes sense making this change, ```
git diff
diff --git a/avocado-setup.py b/avocado-setup.py
index 33d21ee..aa887f0 100644
--- a/avocado-setup.py
+++ b/avocado-setup.py
@@ -204,7 +204,7 @@ def install_repo(path, name):
     :param name: name of the repository
     """
     cmd = "cd %s;make requirements;make requirements-selftests;python setup.py install" % path
-    helper.runcmd(cmd, info_str="Installing repo: %s" % path,
+    helper.runcmd(cmd, info_str="Installing %s from %s" % (name, path),
                   err_str="Failed to install %s repository:" % name)
 
 
@@ -225,7 +225,7 @@ def get_repo(repo, basepath, install=False):
     else:
         cmd = "cd %s;git clone %s %s" % (basepath, repo, repo_name)
         helper.runcmd(cmd,
-                      debug_str="Cloning the repo: %s" % repo_name,
+                      info_str="Cloning the repo: %s in %s" % (repo_name, repo_path),
                       err_str="Failed to clone %s repository:" % repo_name)
     if install:
         install_repo(repo_path, repo_name)
```